packages/safety-suite-api/src/api/dashboardSink/enforcements.ts

```diff
@@ -0,0 +1,56 @@
+import {Request} from '@idelic/safety-net';
+
+import {runApi} from '../../runApi';
+import {ApiOptions, ApiSuccessResponse} from '../../types';
+import {BaseFields, Query, TableQuery} from './types';
+import {convertSortsToStrings} from './util';
+
+export interface EnforcementCounts {
+  inspections: number;
+  cleanInspections: number;
+  inspectionsWithViolations: number;
+}
+
+export interface Enforcement extends BaseFields {
+  recordNumber: number;
+  date: string;
+  csaPoints: number;
+  outOfService: boolean;
```

Comment:
Missing `employeeLabel: string | null`

Reply:
Also missing `id: number` and `customerAlias: string` but I don't think you're using those.

Reply:
`employeeLabel`, `id`, and `customerAlias` are in `BaseFields`.